packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -83,6 +83,11 @@ import { aptosStandardSupportedWalletList } from "./AIP62StandardWallets/registr
 
 export type IAptosWallet = AptosStandardWallet & Wallet;
 
+export interface DappConfig {
```

Comment:
youd need to explicitly export this type so dapps can use it
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/index.ts#L3

we should probably move those types/interfaces to `types.ts` file or something and just export this whole file